### PR TITLE
Refresh Italian localization 2/2

### PR DESCRIPTION
**Pull request by erciccione**
Refreshed present Italian localization

- improved consistency
- fixe typos
- refresh of all the translated pages
- deleted repetitions
- improved fluence of the text
- fixed broken markdown in /it/building.md

This PR is split in two parts. The first part is in kovri-site (ref: monero-project/kovri-site#42)

### CODE REVIEW

File: i18n/it/building.md [hunk 1]
```diff
@@ -143,25 +143,27 @@ Dipendenze opzionali:
 ```bash
 $ sudo pkg_add miniupnpc #For users behind a restrictive NAT
 $ sudo pkg_add doxygen graphviz
+```
 
 # Compilazione ultimo boost
-```bash
+```bash```
```
Comment (anonimal) [inline i18n/it/building.md]: NACK. Breaks the formatting.

Comment (erciccione) [reply i18n/it/building.md]: I don't know what i saw, fixed

File: i18n/it/building.md [hunk 2]
```diff
@@ -143,25 +143,27 @@ Dipendenze opzionali:
...
-$ wget [latest boost] -O latest_boost.tar.bz2
+$ wget [ultimo boost] -O latest_boost.tar.bz2
 $ tar xvjf latest_boost.tar.bz2 && cd latest_boost/
 
 # Scarica e applica le patch
 # https://svn.boost.org/trac/boost/attachment/ticket/12575/boost-1.62-asio-libressl.patch
 # https://gist.githubusercontent.com/laanwj/bf359281dc319b8ff2e1/raw/92250de8404b97bb99d72ab898f4a8cb35ae1ea3/patch-boost_test_impl_execution_monitor_ipp.patch
 
 # Compilazione boost
+```
```
Comment (anonimal) [inline i18n/it/building.md]: Remove this (breaks formatting).

Comment (erciccione) [reply i18n/it/building.md]: fixed and checked all, should be ok now

**Pull request merged**